src/gui/MainWindow.cpp

```diff
@@ -526,12 +526,12 @@ void MainWindow::finalize()
 					m_toolBar );
 	automation_editor_window->setShortcut( Qt::CTRL + Qt::Key_4 );
 
-	ToolButton * fx_mixer_window = new ToolButton(
-					embed::getIconPixmap( "fx_mixer" ),
-					tr( "FX Mixer" ) + " (Ctrl+5)",
-					this, SLOT( toggleFxMixerWin() ),
+	ToolButton * mixer_window = new ToolButton(
+					embed::getIconPixmap( "mixer" ),
```

Comment:
This requires renaming `fx_mixer.png` files under `data/themes/default` and `data/themes/classic` to `mixer.png`, otherwise the icon won't show up. The same for 2 more occurrences.

Reply:
Nevermind, I didn't know they had already been renamed.